### PR TITLE
Stereopair test

### PR DESCRIPTION
**Pull request by michalpelka**
## What does this PR do?

Introduce new level for testing stereopair

## How was this PR tested?

Run demo.

### CODE REVIEW

**Review by jhanca-robotecai — COMMENTED** [review]
Tested stereopair level, tested demolevel1 - all works ass expected.

Some comments in the code; an issue https://github.com/RobotecAI/ROSCon2023Demo/issues/303 not related to changes in this PR created

File: Project/Gem/CMakeLists.txt [hunk 1]
```diff
@@ -86,6 +86,12 @@ ly_add_target(
             Gem::ROS2.Static
 )
 
+find_package(OpenCV REQUIRED)
```
[the remark this answers is not in the document]
Comment (michalpelka) [reply Project/Gem/CMakeLists.txt:89]: To do it fast with mininal code yes. Added opencv as dependency

File: Project/project.json [hunk 2]
```diff
@@ -31,6 +31,8 @@
         "WarehouseAutomation",
         "RobotecWarehouseRacksAssets",
         "RobotecWarehouseBarriersAndCagesAssets",
-        "RobotecWarehouseForkliftsAssets"
+        "RobotecWarehouseForkliftsAssets",
+        "ROS2ScriptIntegration",
+        "Smoothing"
```
[the remark this answers is not in the document]
Comment (michalpelka) [reply Project/project.json:36]: Docker is generally building main, not development. It need to be rectfied in new PR.

File: Project/Assets/Robotec_logo_2022-RGB.svg
[the remark this answers is not in the document]
Comment (michalpelka) [reply Project/Assets/Robotec_logo_2022-RGB.svg:1]: Svg was redundant.

**Pull request merged**